nipype/utils/profiler.py

```diff
@@ -138,14 +137,20 @@ def log_nodes_cb(node, status):
     import logging
     import json
 
+    # Address nested case
+    if isinstance(node.result.runtime, (list,)):
+        resrunlist = node.result.runtime[0]
```

Comment:
`resrunlist` is a single runtime, not a list. Let's just go with `result_runtime` as variable name.